python/gigl/common/services/vertex_ai.py

```diff
@@ -355,3 +468,35 @@ def wait_for_run_completion(
                 f"Vertex AI run stopped with status: {run.state}. "
                 f"Please check the Vertex AI page to trace down the error."
             )
+
+
+def _get_machine_spec(job_config: VertexAiJobConfig) -> MachineSpec:
```

Comment:
```suggestion
def _create_machine_spec(job_config: VertexAiJobConfig) -> MachineSpec:
```

similar for other function